CONTRIBUTING.md

```diff
@@ -64,25 +89,38 @@ Please make sure you label and update your Pull Request accordingly. Alternative
 	- ``git commit`` | Write an informative commit message and save.
 9. Push your changes to your fork
 	- ``git push origin myfeature``
-9. Create a Pull Request -- GitHub will pick up your latest branch committed to automatically when accessing CorsixTH/CorsixTH.
+9. Create a Pull Request -- GitHub will pick up your latest branch committed to
+automatically when accessing CorsixTH/CorsixTH.
 
-The developers will review and report any changes needed to your Pull Request so it could be accepted.
+The developers will review and report any changes needed to your Pull Request
+so it could be accepted.
 
 #### Notes ####
 ##### Multiple commits: #####
-If your feature is very big, break it into smaller steps and do a separate commit for every independent operation.  This means doing step 10 several times as you go. You still only need to do one pull request at the end.
+If your feature is very big, break it into smaller steps and do a separate
+commit for every independent operation.  This means doing step 10 several times
+as you go. You still only need to do one pull request at the end.
 
 ##### Syncing with Upstream: #####
-If it takes a long time between when you start your feature and when you finish there might be other important changes other people are making to CorsixTH.  It is a good idea to make sure your code will still operate correctly with the latest changes.  To do this:
+If it takes a long time between when you start your feature and when you finish
+there might be other important changes other people are making to CorsixTH.  It
+is a good idea to make sure your code will still operate correctly with the
+latest changes.  To do this:
 - Commit your work to the feature branch
 - ``git checkout master``
 - ``git fetch upstream``
 - ``git merge --ff-only master``
-- ``git checkout myfeature`` | switch back to your feature branch to continue working on it
-
-*NB: GitHub Desktop users need to conduct fetch upstream from your branch on github.com and rebase from GitHub Desktop after*\
-What this does is downloads all the changes from CorsixTH/CorsixTH since you started, and pretends that all your changes were made after them.  If there are conflicts, for example if someone else changed the same line in the same file that you did you will be asked to resolve those conflicts.
-
-If you follow these guidelines then you should be well on your way to producing good pull requests.
+- ``git checkout myfeature`` | switch back to your feature branch to continue
+working on it
+
+*NB: GitHub Desktop users need to conduct fetch upstream from your branch on
+github.com and rebase from GitHub Desktop after*\
+What this does is downloads all the changes from CorsixTH/CorsixTH since you
+started, and pretends that all your changes were made after them.  If there are
```

Comment:
replace `and pretends that all your changes were made after them` with `and moves your commits on top of them`

Reply:
wording predates your change - feel free to ignore

Reply:
```suggestion
started, and moves your commits on top of them.  If there are
```